app/models/collection.rb

```diff
@@ -490,4 +491,10 @@ def clear_icon
     self.icon = nil if delete_icon? && !icon.dirty?
   end
 
+  after_save :reindex_collection
```

Comment:
Should we remove these callbacks? The Searchable module set up similar callbacks:

https://github.com/otwcode/otwarchive/blob/63adfaf4641a6fc9c2e24911d86a563fe8867508/lib/searchable.rb#L5-L6